### PR TITLE
Oxidized improvements

### PR DESCRIPTION
**Pull request by murrant**
Import more settings to the UI
use lnms config:set
fix Oxidized API sql injection

image

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: html/js/manifest.js [hunk 1]
```diff
@@ -1 +1,192 @@
...
\ No newline at end of file
+/******/ (() => { // webpackBootstrap
```
Comment (Jellyfrog) [inline html/js/manifest.js:1]: Run in production mode please

**Review by Jellyfrog — CHANGES_REQUESTED** [review]

File: html/js/manifest.js (hunk 1 above)
Comment (murrant) [reply html/js/manifest.js:1]: Sorry, last commit before sleep.  Started implementing maps in the settings.

Comment (Jellyfrog) [reply html/js/manifest.js:1]: :)

**Review by Jellyfrog — APPROVED** [review]

File: app/Models/Device.php [hunk 2]
```diff
@@ -474,15 +485,7 @@ public function scopeWhereUptime($query, $uptime, $modifier = '<')
...
-        return $query->where('disabled', 0)
-            ->leftJoin('devices_attribs', function (JoinClause $query) {
-                $query->on('devices.device_id', 'devices_attribs.device_id')
-                    ->where('devices_attribs.attrib_type', 'override_icmp_disable');
-            })
-            ->where(function (Builder $query) {
-                $query->whereNull('devices_attribs.attrib_value')
-                    ->orWhere('devices_attribs.attrib_value', '!=', 'true');
-            });
+        return $this->scopeWhereAttributeDisabled($query->where('disabled', 0), 'override_icmp_disable');
```
Comment (Jellyfrog) [inline app/Models/Device.php:488]: Not really sure what the convention is when using scope functions internally,
but normally you call it with:
```suggestion
        return $query->where('disabled', 0)->whereAttributeDisabled('override_icmp_disable');
```
Maybe thats prettier?

File: app/Models/Device.php [hunk 3]
```diff
@@ -464,6 +464,17 @@ public function scopeIsNotDisabled($query)
         ]);
     }
 
+    public function scopeWhereAttributeDisabled($query, $attribute)
```
Comment (Jellyfrog) [inline app/Models/Device.php:467]: We should start type-hinting on new functions
```suggestion
    public function scopeWhereAttributeDisabled(Builder $query, string $attribute) : Builder
```

Comment (murrant) [reply app/Models/Device.php:467]: No idea what the convention is.

**Comment by murrant** [comment]
Larastan no likey @Jellyfrog .

**Comment by Jellyfrog** [comment]
> Larastan no likey @Jellyfrog .

Hmm I guess it doesn't support that then, seems it doesn't understand what model it's working on

File: html/mix-manifest.json [hunk 4]
```diff
@@ -1,13 +1,26 @@
...
+    "/js/lang/id.js": "/js/lang/id.js?id=6c3ef97eeb2cd2818885",
     "/js/lang/it.js": "/js/lang/it.js?id=b28a63928155eeb4e2a1",
+    "/js/lang/nl.js": "/js/lang/nl.js?id=052c04cc5d1f408fcf55",
+    "/js/lang/pl.js": "/js/lang/pl.js?id=0cca1dde319e96cac73d",
+    "/js/lang/pt-br.js": "/js/lang/pt-br.js?id=cf9e6e0244e1a17fddf1",
+    "/js/lang/pt.js": "/js/lang/pt.js?id=38b658d49f91c95b2566",
+    "/js/lang/ro.js": "/js/lang/ro.js?id=55032462a25b893fa186",
     "/js/lang/ru.js": "/js/lang/ru.js?id=f6b7c078755312a0907c",
+    "/js/lang/th.js": "/js/lang/th.js?id=3d5d6b9874b3ea673b3f",
+    "/js/lang/tr.js": "/js/lang/tr.js?id=d964e6a3e0bc51bbd7a1",
```
Comment (Jellyfrog) [inline html/mix-manifest.json:23]: where did all these come from?

File: html/css/dark.css [hunk 5]
```diff
@@ -1,6 +1,3 @@
-  body {
-    background: #333;
```
Comment (Jellyfrog) [inline html/css/dark.css:2]: you sure about this?

Comment (murrant) [reply html/css/dark.css:2]: Yes, body background is set further down.  Just removing the stupid unused one :)

File: html/mix-manifest.json (hunk 4 above)
Comment (murrant) [reply html/mix-manifest.json:23]: It's odd, when you run npm it creates hashes for more files than lnms translation:generate.  It is harmless however.  The hash is only used for cache breaking.

**Pull request merged**